nca/NetworkConfig/TopologyObjectsFinder.py

```diff
@@ -217,6 +228,8 @@ def _add_hep_from_yaml(self, hep_object):
             hep.add_profile(profile)
 
         self._add_peer(hep)
+        if ExplTracker().is_active():
+            ExplTracker().add_item(hep_object.path, hep_object.line_number, hep.full_name())
```

Comment:
I am not sure I understand. The display names for the nodes are in the graph itself (svg for example). The explainability needs to describe the policies affecting them and where the configurations are. It can also get, as input, 2 nodes in the format of endpoints=deployments (with the type in Parenthesis).